homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -311,7 +311,8 @@ def preset_mode(self):
     @property
     def available(self):
         """Return true when state is known."""
-        return self._available
+        available = super().available
+        return available and self._available
```

Comment:
Could be simplified to `return super().available and self._available`

Reply:
Sure